### PR TITLE
apps: Replace Atom with Firefox

### LINKED ISSUE #644: Replace Atom in featured apps with a well known app with nice visual identity
https://github.com/flatpak/flatpak.github.io/blob/source/data/apps.yml#L61

https://github.blog/2022-06-08-sunsetting-atom/

/cc @jimmac

### PR DESCRIPTION
**Pull request by AsciiWolf**
Closes #644.

### CODE REVIEW

File: data/apps.yml [hunk 1]
```diff
@@ -58,15 +58,15 @@
...
-- name:     "Atom"
-  id:       "io.atom.Atom"
-  ref:      "app/io.atom.Atom/x86_64/stable"
-  summary:  "A hackable text editor for the 21st Century"
-  color:    "#96CBA5"
-  "flatpakref": "https://flathub.org/repo/appstream/io.atom.Atom.flatpakref"
+- name:     "Firefox"
+  id:       "org.mozilla.firefox"
+  ref:      "app/org.mozilla.firefox/x86_64/stable"
+  summary:  "Fast, Private & Safe Web Browser"
```
Comment (AsciiWolf) [inline data/apps.yml:64]: Not sure if `&` is fine here or whether it should be replaced with `&amp;`.

File: data/apps.yml [hunk 2]
```diff
@@ -58,15 +58,15 @@
...
+  color:    "#FFDA42"
```
Comment (AsciiWolf) [inline data/apps.yml]: @jimmac Is this a good color for the Firefox icon or should I select a different one?

Comment (AsciiWolf) [reply data/apps.yml]: `#FFEF42` also looks good. Is it better than `#FFDA42`?

File: data/apps.yml (hunk 1 above)
Comment (AsciiWolf) [reply data/apps.yml:64]: @razzeee Any idea?

Comment (razzeee) [reply data/apps.yml:64]: I'm not sure if ruby cares. We have multiple of those problems in the `distro.yml` which nobody seemed to care for so far

File: data/apps.yml (hunk 2 above)
Comment (razzeee) [reply data/apps.yml]: Just checked, if they set something in their appdata, unfortunatly not.

https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/docker/firefox-flatpak/org.mozilla.firefox.appdata.xml.in

Comment (AsciiWolf) [reply data/apps.yml]: I wonder whether something can be used to calculate the right color. I picked it by hand.

Comment (jimmac) [reply data/apps.yml]: I don't even remember where it's used.

Comment (AsciiWolf) [reply data/apps.yml]: Well, I think it is not used. :-) I vaguely remember that there used to be a page with featured Flatpak apps on Flatpak.org many, many years ago. This is probably what remained from it. But I would still like to set the color to a correct looking one. :-) I will probably go with `#FFEF42`.

**Comment by AsciiWolf** [comment]
Let's merge this. :-)

**Pull request merged**